src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+public class Task {
+    private String task;
+    private boolean isDone;
+
+
+    public Task(String task, boolean isDone) {
+        this.task = task;
+        this.isDone = isDone;
+    }
+
+    public Task() {
+        this(null, false);
+    }
+
+    public String getTask() {
```

Comment:
The confusion here caused bu the name of the variable name "task: String". The method named "getTask()" in the class "Task" should return the object of the class, which is not the case here.